solutions/swb-app/src/generateRouter.ts

```diff
@@ -50,14 +51,21 @@ export function generateRouter(apiRouteConfig: ApiRouteConfig): Express {
   app.use(cookieParser());
   app.use(csurf('none'));
 
+  const tokenRevocationService: TokenRevocationService = new TokenRevocationService({
+    dynamoDBSettings: {
+      region: process.env.AWS_REGION!,
+      table: process.env.REVOKED_TOKENS_DDB_TABLE_NAME!
+    }
+  });
   const cognitoPluginOptions: CognitoAuthenticationPluginOptions = {
     cognitoDomain: process.env.COGNITO_DOMAIN!,
     userPoolId: process.env.USER_POOL_ID!,
     webUiClient: {
       clientId: process.env.WEB_UI_CLIENT_ID!,
       clientSecret: process.env.WEB_UI_CLIENT_SECRET!
     },
-    allowedClientIds: [process.env.PROGRAMMATIC_ACCESS_CLIENT_ID!]
+    allowedClientIds: [process.env.PROGRAMMATIC_ACCESS_CLIENT_ID!],
+    tokenRevocationService
```

Comment:
can you double check if the conflict is resolved in the expected way. it doesn't match current release branch

Reply:
Confirmed that this is from updates made on the develop branch.